configs/cms-xcache/config.d/52-cms-xcache-lcmaps-authz.cfg

```diff
@@ -0,0 +1,7 @@
+sec.protocol /usr/lib64 gsi -certdir:/etc/grid-security/certificates -cert:/etc/grid-security/xrd/xrdcert.pem -key:/etc/grid-security/\
+xrd/xrdkey.pem -crl:1 -authzfun:libXrdLcmaps.so -authzfunparms:--lcmapscfg,/etc/xrootd/lcmaps.db,--loglevel,5 -gmapopt:10 -gmapto:0
```

Comment:
Please split this into more lines like https://github.com/opensciencegrid/xcache/blob/master/configs/stash-cache/config.d/50-stash-cache-authz.cfg#L27-L37